modules/libpng/1.6.44/overlay/BUILD.bazel

```diff
@@ -0,0 +1,76 @@
+# Copied from https://github.com/google-deepmind/lab2d/blob/09474439959534b1de3fce3d4a4ed35d565ea230/bazel/png.BUILD
```

Comment:
Did this change? If not, could it be symlinked from the previous version?

Reply:
Switched from patches to overlay (since this is easier to maintain) - no symlink possible here